scripts/clan.py

```diff
@@ -457,6 +463,9 @@ def save_clan(self):
             clan_data["med_cat"] = None
         clan_data["med_cat_number"] = self.med_cat_number
         clan_data["med_cat_predecessors"] = self.med_cat_predecessors
+        
+        if self.custom_pronouns:
+            clan_data["custom_pronouns"] = self.custom_pronouns
```

Comment:
There is no need for lines 467 and 468. self.custom_pronouns is already assigned to the key "custom_pronouns" on line 439.

Reply:
Deleted 👍